en/SUMMARY.md

```diff
@@ -68,6 +68,9 @@
       * [Omnicopter](frames_multicopter/omnicopter.md)
       * [DJI F450 (CUAV v5+)](frames_multicopter/dji_f450_cuav_5plus.md)
       * [DJI F450 (CUAV v5 nano)](frames_multicopter/dji_f450_cuav_5nano.md)
+    * [Features](features_mc/README.md)
+      * [Throw Launch](features_mc/throw_launch.md)
+      * [Precision Landing](advanced_features/precland.md)
```

Comment:
I'd be happy for them to be modes when they are - indeed I tried to push precland into being a mode recently. But it's hard because it can happen inside a return or mission or forced through shell - but not standalone. Just odd. 

Are there any other features that we might consider MC only that don't depend on Computer vision - other than collision prevention of course?

Reply:
PS I moved the document for throw launch into flight_modes_mc folder, which will affect the URL. However it will still be displayed as an mc feature until it is really a flight mode. This means it won't break later when it is a mode. 
Note that we will be grouping the modes together under MC (except this one) soon. I just have to finish splitting Return mode and mission mode.